*** Starting uWSGI 2.0.18 (64bit) on [Mon Sep  2 00:44:49 2019] ***
compiled with version: 7.4.0 on 01 September 2019 16:38:37
os: Linux-5.0.0-25-generic #26~18.04.1-Ubuntu SMP Thu Aug 1 13:51:02 UTC 2019
nodename: ubuntu
machine: x86_64
clock source: unix
detected number of CPU cores: 4
current working directory: /home/moluo/Desktop/luffycity/luffyapi
writing pidfile to uwsgi.pid
detected binary path: /home/moluo/.virtualenvs/luffy/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/moluo/Desktop/luffycity/luffyapi
your processes number limit is 11710
your memory page size is 4096 bytes
detected max file descriptor number: 1048576
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 127.0.0.1:8000 fd 3
Python version: 3.6.8 (default, Jan 14 2019, 11:02:34)  [GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]
Set PythonHome to /home/moluo/.virtualenvs/luffy
Python main interpreter initialized at 0x555b44c66a60
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416720 bytes (406 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
Traceback (most recent call last):
  File "luffyapi/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
  File "/home/moluo/.virtualenvs/luffy/lib/python3.6/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/home/moluo/.virtualenvs/luffy/lib/python3.6/site-packages/django/__init__.py", line 19, in setup
    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
  File "/home/moluo/.virtualenvs/luffy/lib/python3.6/site-packages/django/conf/__init__.py", line 79, in __getattr__
    self._setup(name)
  File "/home/moluo/.virtualenvs/luffy/lib/python3.6/site-packages/django/conf/__init__.py", line 66, in _setup
    self._wrapped = Settings(settings_module)
  File "/home/moluo/.virtualenvs/luffy/lib/python3.6/site-packages/django/conf/__init__.py", line 176, in __init__
    raise ImproperlyConfigured("The SECRET_KEY setting must not be empty.")
django.core.exceptions.ImproperlyConfigured: The SECRET_KEY setting must not be empty.
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 33107)
spawned uWSGI worker 1 (pid: 33109, cores: 2)
spawned uWSGI worker 2 (pid: 33110, cores: 2)
spawned uWSGI worker 3 (pid: 33112, cores: 2)
spawned uWSGI worker 4 (pid: 33113, cores: 2)
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.18 (64bit) on [Mon Sep  2 00:48:55 2019] ***
compiled with version: 7.4.0 on 01 September 2019 16:38:37
os: Linux-5.0.0-25-generic #26~18.04.1-Ubuntu SMP Thu Aug 1 13:51:02 UTC 2019
nodename: ubuntu
machine: x86_64
clock source: unix
detected number of CPU cores: 4
current working directory: /home/moluo/Desktop/luffycity/luffyapi
writing pidfile to uwsgi.pid
detected binary path: /home/moluo/.virtualenvs/luffy/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/moluo/Desktop/luffycity/luffyapi
your processes number limit is 11710
your memory page size is 4096 bytes
detected max file descriptor number: 1048576
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 127.0.0.1:8000 fd 3
Python version: 3.6.8 (default, Jan 14 2019, 11:02:34)  [GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]
Set PythonHome to /home/moluo/.virtualenvs/luffy
Python main interpreter initialized at 0x5615c5ba1a60
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416720 bytes (406 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
Traceback (most recent call last):
  File "luffyapi/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
  File "/home/moluo/.virtualenvs/luffy/lib/python3.6/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/home/moluo/.virtualenvs/luffy/lib/python3.6/site-packages/django/__init__.py", line 19, in setup
    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
  File "/home/moluo/.virtualenvs/luffy/lib/python3.6/site-packages/django/conf/__init__.py", line 79, in __getattr__
    self._setup(name)
  File "/home/moluo/.virtualenvs/luffy/lib/python3.6/site-packages/django/conf/__init__.py", line 66, in _setup
    self._wrapped = Settings(settings_module)
  File "/home/moluo/.virtualenvs/luffy/lib/python3.6/site-packages/django/conf/__init__.py", line 176, in __init__
    raise ImproperlyConfigured("The SECRET_KEY setting must not be empty.")
django.core.exceptions.ImproperlyConfigured: The SECRET_KEY setting must not be empty.
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 33339)
spawned uWSGI worker 1 (pid: 33341, cores: 2)
spawned uWSGI worker 2 (pid: 33342, cores: 2)
spawned uWSGI worker 3 (pid: 33343, cores: 2)
spawned uWSGI worker 4 (pid: 33344, cores: 2)
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
